apps/modernization-ui/src/apps/page-builder/pages/QuestionLibrary/QuestionLibraryTable.tsx

```diff
@@ -126,22 +112,8 @@ export const QuestionLibraryTable = ({ summaries, pages, qtnModalRef }: Props) =
     };
 
     const handleAddQsntoPage = async () => {
-        const request = {
-            orderNumber: subsectionId,
-            questionId: selectedQuestion.id
-        };
-        history.go(-1);
-
-        PageQuestionControllerService.addQuestionToPageUsingPost({
-            authorization,
-            page: Number(pageId),
-            request
-        }).then((response: any) => {
-            setSelectedQuestion({});
-            showAlert({ type: 'success', header: 'Add', message: 'Add Question successfully on page' });
-            history.go(-1);
-            return response;
-        });
+        onAddQuestion && onAddQuestion(Number(selectedQuestion.id));
+        setSelectedQuestion({});
```

Comment:
Question: Doesn't this table allow multiple questions to be selected? If so, i think this will need to be an array.

Reply:
Hi Mike
If I remember it correctly, you have this branch locally and you were asking me to do it for single selection. So that you can take the code and try in your local for multiple selection. Please correct if any concerns.

Reply:
you are very much correct. Ty.